pact/coin-contract/v4/coin-v4.pact

```diff
@@ -18,9 +18,19 @@
     ]
 
   (implements fungible-v2)
+<<<<<<< HEAD
```

Comment:
Are the git conflict markers supposed to be here?

Reply:
The file `coin-contract/v4/coin-v4.pact` seems to be syntactically invalid. But all tests for this revision of the PR are green. Does that mean that there is no unit test coverage for coin contract V4?

I propose that before fixing the contract code, a unit test that catches this should be added, to make sure that there is  coverage.

Reply:
This is still WIP